### PR TITLE
git-mv: add page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

File: pages/common/git-mv.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# git mv
+
+> Move or rename files and update the git index.
+
+- Rename or move file and add it to next commit
```
Comment (waldyrious) [inline pages/common/git-mv.md]: I'd make separate examples for moving and renaming, with the moving example including a relative path in the filename parameter (and no "new" prefix), to make the distinction clearer.

File: pages/common/git-mv.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`git mv {{filename}} {{new_filename}}`
+
+- Rename or move file, even if target exists
```
Comment (waldyrious) [inline pages/common/git-mv.md]: I would prefer this to explicitly say "overwrite the file in the target path if it exists", to ensure people understand what they're doing.

Comment (igorshubovych) [inline pages/common/git-mv.md]: So do you agree with me?

**Comment by ostera** [comment]
:+1: if we make the change suggested by @waldyrious

File: pages/common/git-mv.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/git-mv.md]: Yes, it's just a change in the phrasing, to make the message more explicit, but the message is the same :)

**Pull request merged**